src/components/space/ActionsChart.js

```diff
@@ -32,12 +78,12 @@ function ActionsChart() {
       </Typography>
       <ResponsiveContainer width="95%" height={450}>
         <BarChart
-          data={actionsByDay}
+          data={formattedActionsByDay}
           margin={{ top: 30, bottom: 20, left: 20, right: 20 }}
         >
           <CartesianGrid strokeDasharray="2" />
-          <XAxis dataKey="date" />
-          <YAxis />
+          <XAxis dataKey="date" tick={{ fontSize: 14 }} />
```

Comment:
Can we factor out these styling bits?

Reply:
I experimented with this but Recharts does not respond well to such changes, so will leave as is (at least for now).